spec/rubocop/cop/style/hash_except_spec.rb

```diff
@@ -225,6 +261,18 @@
       RUBY
     end
 
+    it 'does not register an offense when using more than two block arguments' do
+      expect_no_offenses(<<~RUBY)
+        {foo: 1, bar: 2, baz: 3}.reject { |k, v, o| k == :bar }
+      RUBY
+    end
```

Comment:
Technically, this is something this cop could handle:
```ruby
{foo: 1, bar: 2, baz: 3}.reject { |k, v, o| k == :bar } == {foo: 1, bar: 2, baz: 3}.except(:bar)
# => true
```
but I think it's out of scope of this PR.